crates/autopilot/src/on_settlement_event_updater.rs

```diff
@@ -0,0 +1,342 @@
+//! This module is responsible for updating the database, for each settlement
```

Comment:
I really with the file rename were properly rendered as such. Would make it much easier to see the difference here. I would be in favour of not renaming the file until later so see this diff better.

Reply:
In this case I believe it's fine to look at whole file, because I changed everything except `run_forever` function.